### PR TITLE
#139456259 Start a game and save records

### PR DESCRIPTION
**Pull request by iAmao**
#### What does this PR do?

Allows user to click a button to start a game and save game records

#### Description of Task to be completed?

User sees a 'play' button on successful login,
when user clicks the button a modal pops up and then,
user gets the options of either playing with a friends or strangers.
Game saves once the user start game and updates after each round

#### How should this be manually tested?

- clone the repo : `git clone https://github.com/andela/izanami-cfh.git`
- install dependencies: `npm install`
- start the app: `npm start`

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#139456259 Users should be able to create/start a new game

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A

### CODE REVIEW

**Review by eshaibu — APPROVED** [review]

File: config/routes.js [hunk 1]
```diff
@@ -104,5 +104,7 @@ module.exports = (app, passport) => {
   const index = require('../app/controllers/index');
   app.get('/play', index.play);
```
Comment (ooduye) [inline config/routes.js:105]: Get rid of this

File: config/socket/game.js [hunk 2]
```diff
@@ -224,13 +227,15 @@ Game.prototype.stateResults = function (self) {
...
+  this.updateGame(self.players);
   for (let i = 0; i < self.players.length; i++) {
     if (self.players[i].points >= self.pointLimit) {
       winner = i;
     }
   }
-  self.sendUpdate();
-  self.resultsTimeout = setTimeout(() => {
+
+  // self.sendUpdate(self.players);
```
Comment (ooduye) [inline config/socket/game.js]: Remove console log

File: public/css/style.css [hunk 3]
```diff
@@ -1249,6 +1249,16 @@
    }
  }
 
+#play-game-modal {
+    text-align: center;
+}
+
+/* start a game modal styling */
+#play-game-modal-content {
+    margin-top: 100px;
```
Comment (ooduye) [inline public/css/style.css:1289]: Fix indentation from 4 to 2 spaces

File: config/socket/game.js [hunk 4]
```diff
@@ -449,4 +454,46 @@ Game.prototype.drawCard = function () {
...
+  const newGame = new GameModel({
+    created_by: game_players[0],
+    number_of_players: game_players.length.toString(),
+    game_id: players[0].socket.gameID,
+    players: game_players
+  });
+  
+  newGame.save( function( err ){
+    if(!err){
+      console.log('[game.js line 451] game has been saved');
```
Comment (ooduye) [inline config/socket/game.js]: Remove console log and change to comments instead

**Pull request merged**